components/Asana-Card/AsanaCard.js

```diff
@@ -13,12 +19,16 @@ export default function AsanaCard({ name, img, id }) {
         <Link href={`/asanas/${id}`} passHref>
           <StyledLink>Mehr Infos</StyledLink>
         </Link>
+        {showDeleteButton && (
+          <StyledDeleteButton onClick={deleteCard}>X</StyledDeleteButton>
```

Comment:
This button does not have an accessible text other than X

Reply:
That's true. I will change that.